context.go

```diff
@@ -123,3 +123,34 @@ func SetHTTPResponseHeader(ctx context.Context, key, value string) error {
 
 	return nil
 }
+
+// AddHTTPResponseHeader adds an HTTP header key-value pair using a context
+// provided by a twirp-generated server, or a child of that context.
+// The server will include the header in its response for that request context.
+//
+// This can be used to respond with custom HTTP headers like "Cache-Control".
+// But note that HTTP headers are a Twirp implementation detail,
+// only visible by middleware, not by the clients or their responses.
+//
+// The header will be ignored (noop) if the context is invalid (i.e. using a new
+// context.Background() instead of passing the context from the handler).
+//
+// If called multiple times with the same key, it appends to any existing values
+// associated with that key.
+//
+// AddHTTPResponseHeader returns an error if the provided header key
+// would overwrite a header that is needed by Twirp, like "Content-Type".
+func AddHTTPResponseHeader(ctx context.Context, key string, values ...string) error {
```

Comment:
change variadic for simply `value string`. The standard Header method also uses one value, if you want to add multiple values, call the method multiple times.

Reply:
done 👍 
https://github.com/twitchtv/twirp/pull/227/commits/c3c92a53c49b5dbdd775590580e3220cdbdd8e81

Reply:
done 👍
https://github.com/twitchtv/twirp/pull/227/commits/c3c92a53c49b5dbdd775590580e3220cdbdd8e81